src/app/public/src/modules/wrapper/wrapper.component.ts

```diff
@@ -8,9 +8,11 @@ import { Subscription } from 'rxjs';
 
 import { StacheTitleService } from './title.service';
 import { StachePageAnchorComponent } from '../page-anchor';
-import { StacheJsonDataService } from '../shared';
+import { StacheConfigService, StacheJsonDataService } from '../shared';
 import { StacheNavLink, StacheNavService } from '../nav';
 
+const _get = require('lodash.get');
```

Comment:
Non-blocking feedback:

Adding a new dependency gives me pause, because it 1) adds a feature that we must monitor to prevent introducing a future breaking change, and 2) it increases the risk for transitive dependencies.

In this case, I don't think the cost-benefit of the dependency is merited; I know it offers a lot of convenience for this specific use-case, but I'd rather write out the long-form `if` statement and reduce the risks mentioned.

Reply:
I'm not sure I agree in this case. This particular method helps eliminate a ton of annoying, repetitive checks in a consistent way. I could see pulling this out into a utils function so that it's centralized, but we constantly do this kind of check. The package itself is small, self contained, and well supported and tested. If you really object that strongly to a dependency, I could see recreating it in a service too instead, but it seems simpler to just pull this in and use it. Thoughts?

Reply:
Sounds good. As long as every other instance where we're doing the long `if` checks are replaced with whatever you decide. Again, non-blocking for this pull request.